src/targets/gpu/compile_hip_code_object.cpp

```diff
@@ -153,29 +153,35 @@ std::size_t compute_block_size(std::size_t n, std::size_t max_block_size)
     return std::min(std::max(min_block_size, block_size), max_block_size);
 }
 
+#ifndef _WIN32
+std::vector<src_file> migraphx_kernels()
+{
+    auto kernels{::migraphx_kernels()};
+    std::vector<src_file> result;
+    std::transform(
+        kernels.begin(),
+        kernels.end(),
+        std::back_inserter(result),
+        [](std::pair<std::string_view, std::string_view> const& elem) { return src_file{elem}; });
+    return result;
+}
+#endif
```

Comment:
Windows should be calling the same function. This `ifdef` should be removed.

Reply:
We are not translating on Windows from `pair` to `src_file` - the function `migraphx_kernels` returns `std::vector<src_file>` for better optimizations, i.e., RVO since this data is already in memory and desired format.

Reply:
The function generated by `add_embed_library` should use the same signature.